web/packages/teleport/src/Integrations/Enroll/AwsOidc/AwsOidc.tsx

```diff
@@ -178,23 +104,33 @@ export function AwsOidc() {
       <Validation>
         {({ validator }) => (
           <>
-            <Container mb={5}>
+            <Container mb={5} width={800}>
```

Comment:
Width added to make all the components share consistent width.

Previously: 
<img width="1089" alt="Screenshot 2024-10-04 at 12 16 52 PM" src="https://github.com/user-attachments/assets/a4b6b443-28f8-4d3f-b590-a50a6972dd4e">

Updated:
<img width="906" alt="Screenshot 2024-10-04 at 12 17 08 PM" src="https://github.com/user-attachments/assets/1f953a54-08f5-4600-a8e9-3237b91e8abf">